packages/yoga/src/AutoComplete/web/AutoComplete.jsx

```diff
@@ -200,32 +222,43 @@ const AutoComplete = React.forwardRef(
               second.toLowerCase().indexOf(inputValue)
                 ? -1
                 : 1,
-            )
-            .slice(0, 6);
+            );
 
-          const hasSuggestion = isOpen && Boolean(suggestionList.length);
+          if (!!inputValue && isOpen) {
+            setIsSuggestionsOpen(true);
+          }
 
           return (
             <Wrapper
               className={className}
               style={style}
               full={full}
-              isOpen={hasSuggestion}
+              isOpen={isSuggestionsOpen}
               {...getRootProps()}
               ref={ref}
+              onBlur={handleCloseSuggestions}
             >
               <Input
                 {...props}
                 full={full}
                 onClean={cleanable => {
                   onClean(cleanable);
                   clearSelection();
+                  handleCloseSuggestions();
                 }}
                 {...getInputProps({
                   onFocus: () => (inputValue.length ? openMenu() : null),
                 })}
+                rightIcon={
+                  isSuggestionsOpen ? (
+                    <ChevronUp onClick={handleCloseSuggestions} />
```

Comment:
How about adding an aria-label here too? The same you're doing with the `Close` icon

Reply:
Miss it! 😅 Fixed 😁